### DISCUSSION TITLE
Self-contained publish error for wpf application

### DISCUSSION THREAD

**Discussion by lavahasif** [unoplatform/uno#10636] [Q&A]
ide
visual studio 2022 17.4 

log

02/Dec/2022 10:21
System.AggregateException: One or more errors occurred. ---> Microsoft.WebTools.Shared.Exceptions.WebToolsException: Publish has encountered an error. We were unable to determine the cause of the error. Check the output log for more details.
--- End of inner exception stack trace ---
---> (Inner Exception #0) Microsoft.WebTools.Shared.Exceptions.WebToolsException: Publish has encountered an error. We were unable to determine the cause of the error. Check the output log for more details. <---

Microsoft.WebTools.Shared.Exceptions.WebToolsException: Publish has encountered an error. We were unable to determine the cause of the error. Check the output log for more details.

**Comment by jeromelaban** [comment]
Please do not open multiple discussions about the same topic. Let's continue in https://github.com/unoplatform/uno/discussions/10435
